tests/MongoDB.Driver.Core.TestHelpers/CoreTestConfiguration.cs

```diff
@@ -81,7 +81,7 @@ public static SemanticVersion ServerVersion
             get
             {
                 var server = __cluster.Value.SelectServer(WritableServerSelector.Instance, CancellationToken.None);
-                return server.Description.Version;
+                return Ensure.IsNotNull(server.Description.Version, nameof(server.Description.Version));
```

Comment:
`Ensure` is only valid to use for checking arguments to a method.

Either leave this line alone (the caller will either check for null or throw a `NullReferenceException`, or replace this line with:

    var version = server.Description.Version;
    if (version == null)
    {
        throw new InvalidOperationException("ServerDescription.Version is unexpectedly null.");
    }
    return version;